.github/workflows/container-validation-backends.yml

```
@@ -214,6 +215,73 @@ jobs:
+        if: ${{ matrix.platform.arch != 'arm64' }}
+        uses: ./.github/actions/pytest
+        with:
+          image_tag: ${{ steps.build-image.outputs.image_tag }}
+          pytest_marks: "pre_merge and vllm and gpu_0"  # GH arm runners have no GPUs
+          framework: "vllm"
+          test_type: "pre_merge"
+          platform_arch: ${{ matrix.platform.arch }}
+
+  vllm:
+    needs: changed-files
+    if: needs.changed-files.outputs.has_code_changes == 'true' || needs.changed-files.outputs.vllm == 'true'
```

actionlint (1.7.9)

property "has_code_changes" is not defined in object type {core: string; deploy: string; operator: string; sglang: string; trtllm: string; vllm: string}

(expression)